content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+---
+reviewers:
+- gjtempleton
+- jonathan-innis
+- maciekpytel
+title: Node Autoscaling
+linkTitle: Node Autoscaling
+description: >-
+  Automatically provision and consolidate the Nodes in your cluster to adapt to demand and optimize cost.
+content_type: concept
+weight: 130
+---
+
+In order to run workloads in your cluster, you need {{< glossary_tooltip text="Nodes" term_id="node" >}}.
+Nodes in your cluster can be _autoscaled_ - dynamically
+[_provisioned_](#provisioning), or [_consolidated_](#consolidation) to provide needed capacity
+while optimizing cost. Autoscaling is performed by Node [_autoscalers_](#autoscalers).
+
+## Node provisioning {#provisioning}
+
+If there are Pods in a cluster that can't be scheduled on existing Nodes, new Nodes
+can be automatically added to the cluster - _provisioned_ - to accommodate the Pods.
+This is especially useful if the number of Pods changes over time, for example
+as a result of [combining workload and Node autoscaling](#multi-scaling).
+
+Autoscalers provision the Nodes by creating and deleting cloud provider Virtual Machines (VMs) backing them.
+
+The goal of provisioning is to make all Pods schedulable. This goal is not always attainable because of various limitations
+(for example reaching configured provisioning limits, provisioning configuration not being compatible
+with a particular set of pods, lack of cloud provider capacity).
+
+{{< note >}}
+Only the scheduling constraints of pods (for example resource requests, node selectors) are taken into account when determining Nodes to
+provision. The real resource usage of the Pods after they start running is not taken into account. In order
+to autoscale Nodes based on actual resource usage, you can [combine workload and Node autoscaling](#multi-scaling).
+{{< /note >}}
+
+Depending on the choice of autoscaler, provisioning can also be triggered by
```

Comment:
I think minimum (or maximum) node thresholds are more of an autoscaler configuration rather than a required behavior of a particular autoscaler. So I'd probably replace "Depending on the choice of autoscaler..." with

"Autoscaler configuration may also include other node provisioning triggers (for example the number of Nodes falling below a configured minimum limit)."